docs/ldml/tr35-personNames.md

```diff
@@ -67,39 +67,41 @@ The LDML specification is divided into the following parts:
 * 6 [Formatting Process](#formatting-process)
   * 6.1 [Derive the name locale](#derive-the-name-locale)
   * 6.2 [Derive the formatting locale](#derive-the-formatting-locale)
-      * 6.2.1 [Switch the formatting locale if necessary]("switch-formatting-locales)
+    * 6.2.1 [Switch the formatting locale if necessary](#switch-formatting-locales)
   * 6.3 [Derive the name order](#derive-the-name-order)
   * 6.4 [Choose a personName](#choose-a-personname)
-  * 6.5 [Choose a namePattern](#choose-a-namepattern")
-  * 6.6 [Access PersonName object](#access-personname")
-      * 6.6.1 [Handle missing surname](#missing-surname")
-      * 6.6.2 [Handle core and prefix](#core-and-prefix)
-      * 6.6.3 [Derive initials](#derive-initials)
+  * 6.5 [Choose a namePattern](#choose-a-namepattern)
+  * 6.6 [Access PersonName object](#access-personname-object)
+    * 6.6.1 [Handle missing surname](#missing-surname)
+    * 6.6.2 [Handle core and prefix](#core-and-prefix)
+    * 6.6.3 [Derive initials](#derive-initials)
   * 6.7 [Process a namePattern](#process-the-namepattern)
-      * 6.7.1 [Handling foreign names](#handling-foreign-names)
-      * 6.7.2 [Setting the spaceReplacement](#spaceReplacement)
-      * 6.7.3 [Examples of space replacement](#spaceReplacement-examples)
+    * 6.7.1 [Handling foreign names](#handling-foreign-names)
+    * 6.7.2 [Setting the spaceReplacement](#spaceReplacement)
+    * 6.7.3 [Examples of space replacement](#examples-examples)
```

Comment:
Generally good fixes. This link is odd; probably from a mistake further down in the file, but best to fix here.

Reply:
Should be spaceReplacement-examples in 2 places

Reply:
fixed PTAL